packages/ui/v2/core/Shell.tsx

```diff
@@ -742,59 +738,59 @@ function SideBarContainer() {
 
 function SideBar() {
   return (
-    <aside className="desktop-transparent hidden w-14 flex-col border-r border-gray-100 bg-gray-50 md:flex lg:w-56 lg:flex-shrink-0 lg:px-4">
-      <div className="flex h-0 flex-1 flex-col overflow-y-auto pt-3 pb-4 lg:pt-5">
-        <header className="items-center justify-between md:hidden lg:flex">
+    <div className="relative">
```

Comment:
Main change here is that everything is wrapped in this relative div. Besides that you see additional changes on the new lines 771 and 772 highlighted in darker green/red.